docs/user/reference/cdn.rst

```diff
@@ -0,0 +1,39 @@
+Content Delivery Network (CDN) and caching
+==========================================
+
+A CDN is used for making documentation pages *fast* for your users.
+This is done by caching the documentation page content in multiple data centers around the world,
+and then serving docs from the data center closest to the user.
```

Comment:
```suggestion
CDNs increase speed by caching documentation content in multiple data centers around the world,
and then serving docs from the data center closest to the user.
```

Reply:
Shows the value of a second edit! It's hard to get language right on the first pass.